src/js/covidMap.js

```diff
@@ -257,14 +297,33 @@ fluid.defaults("fluid.covidMap.map", {
             source: "matchedRows",
             func: "fluid.transforms.setMembershipToArray"
         },
-        resultsShowing: {
+        citiesPanelShowing: {
+            source: "resultsShowing",
+            target: "dom.citiesList.visible",
+            func: x => !x
+        },
+        resultsPanelShowing: {
             source: "resultsShowing",
             target: "dom.resultsPage.visible"
         },
-        citiesShowing: {
+        onlyShowResultsPanelOnMobile: {
             source: "resultsShowing",
-            target: "dom.citiesList.visible",
-            func: x => !x
+            target: "visiblePanelOnMobileFlags",
+            func: x => {
```

Comment:
See below for rationalising these as modelListeners

Reply:
This comment is still unaddressed

Reply:
I think this model relay is different from other ones such as [`expandCitiesList`](https://github.com/inclusive-design/covid-data-monitor/blob/8422fd6f800c4365fe222c2b9e2c29ffc58bb40c/src/js/covidMap.js#L531-L536) for the location button that can be simplified using modelListeners with `excludeSource: "init"` as the location button doesn't need to handle a false scenario once the initial model change is skipped.

This model relay is controlled by `resultsShowing`, a boolean value that is flipped back and forth when a user interacts with UI. This function ensures the result panel is only shown when `resultsShowing === true` and nothing changes when `resultsShowing === false`. Even if rewriting it into a model listener, the same check and the same function remain without any simplification.

Please let me know if I misunderstood your point. Thanks.